### DISCUSSION TITLE
Template projects for popular tech stacks

### DISCUSSION THREAD

**Discussion by Rolands-Laucis** [Rolands-Laucis/Socio#13] [Ideas]
Starter template project repos for various front-end and back-end tech stacks with a Socio setup ready. Or even as examples.

**Comment by BorisOptrx** [comment]
I just happened to find Socio and I think it has loads of potential. I think the examples for certain common tasks like authentication and perhaps using redis/upstash for token storage, would be very very helpful.
Let me know your thoughts.

**Reply by Rolands-Laucis** [reply]
Yea certainly would be useful. I just dont have a wide experience with web dev tools and frameworks. Vanilla and SvelteKit with relational DBs are my speciality, so for those ive created materials as much as i can. I also now have a react Next.js collab project that uses Socio, so i might include a demo project for that setup at some point as well. But largely this task needs help from other generous people.
Feel free to create starter project repos, if you want, and i can link to them in this repo.
